src/firefly/js/ui/NewImageSearchPanel.jsx

```diff
@@ -22,8 +22,7 @@ import WebPlotRequest, {WPConst} from '../visualize/WebPlotRequest.js';
 import {dispatchPlotImage} from '../visualize/ImagePlotCntlr.js';
 import {imageMasterData} from '../data/ImageMasterData.js';
```

Comment:
@loitly Currently this is a constant, i guess this has to be changed to call server side and return it, right?

Reply:
Nevermind, i've found the call: AllImageSearchConfig.js.